changelog/fragments/run-packagemanifests-flags-2.yaml

```diff
@@ -0,0 +1,15 @@
+entries:
+  - description: Removed the `--olm-namespace` flag from `run packagemanifests`.
```

Comment:
+1 to using the past tense, `Removed`, it reads better when the changelog is generated.